src/routes/productsRoute.js

```diff
@@ -3,7 +3,8 @@ const productsController = require("../controllers/productsController");
 
 const router = express.Router();
 
-router.get("", productsController.getAllProducts);
+router.get('', productsController.getAllProducts);
+router.get('/:productsId', productsController.getProduct);
```

Comment:
복수 표현보다는 단수 표현이 조금 더 적절할 것 같습니다.
```suggestion
router.get('/:productId', productsController.getProduct);
```